cv_next/app/global.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Database as DB } from "@/types/database.types"
+
+declare global {
+  type Database = DB
+  type CvModel = DB["public"]["Tables"]["cvs"]["Row"]
```

Comment:
I'm not a 100% sure models should be defined in a global d.ts file. Especially not one like PaginatedCvsModel. These should be in a models file/folder

Reply:
if its here its automatically picked up by eslint and vscode, and you dont have to import it every time. i saw it in an example somewhere and it seemed to me like the right place but we can talk about it if you're unconvinced

Reply:
resolving for now

Reply:
Don't have to import it doesn't necessarily mean good practice. For me it means I don't know where it's defined..